.github/renovate.json

```diff
@@ -1,11 +1,19 @@
 {
-  "enabledManagers": ["helm-requirements","gradle-wrapper","terraform"],
-  "labels": ["dependencies"],
-  "helm-requirements":
-  {
-    "fileMatch": ["\\Chart.yaml$"],
-    "aliases": {
-      "hmctspublic": "https://hmctspublic.azurecr.io/helm/v1/repo/"
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["local>hmcts/.github:renovate-config"],
+  "labels": ["Renovate-dependencies"],
+  "major": {
+    "dependencyDashboardApproval": true
+  },
+  "packageRules": [
+    {
+      "matchUpdateTypes": [
+        "minor", "patch"
```

Comment:
The .dependabot config should be removed if renovate is being enabled in this way see the conversation in :
#ccd-rpo https://hmcts-reform.slack.com/archives/C03FPH1FH9T/p1695633402641839